hotel-management/src/hooks/room/useFetchRooms.ts

```diff
@@ -1,41 +1,43 @@
 import { useEffect, useState } from 'react';
 
-import { API_ENDPOINT } from '@/constants';
+import { API_ENDPOINT, ROOM_MESSAGE } from '@/constants';
 import HttpService from '@/services/httpService';
-import { Room, ApiResponse } from '@/types';
+import { Room, ApiResponse, RoomParams } from '@/types';
 
 export function useFetchRooms() {
   const [rooms, setRooms] = useState<Room[]>([]);
   const [error, setError] = useState<string | null>(null);
-  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
```

Comment:
The default value of isLoading should be set to false. This is because when the component initially renders, no request has been made yet, so it is not in a loading state. The loading state should only be set to true when the data fetching process begins.